src/components/Layout/LayoutHeader.tsx

```diff
@@ -49,6 +56,16 @@ export const LayoutHeader = ({
       // For keyboard navigators, move focus back to menu button in header
       menuButtonRef?.current?.focus();
```

Comment:
Looks like this isn't working correctly with these changes. When I test keyboard closing the menu, it now focuses the table of contents toggle instead of the menu toggle. 

<img src="https://github.com/user-attachments/assets/2ff6ba87-8d50-4682-bfab-e0cce4189d33" width="400" />